tests/SmimeSigningCertificateTest.php

```diff
@@ -101,7 +101,12 @@ public function testKeyPassphraseIsEncrypted(): void
         $sqlQuery->setFrom('SmimeSigningCertificate');
         $sqlQuery->addWhere(['ID = ?' => $certificate->ID]);
         $result = $sqlQuery->execute();
-        $signingPasswordAsStored = $result->first()['SigningPassword'];
+        $iterator = $result->getIterator();
```

Comment:
`$result->first()` has been replaced with `getIterator` in SS5 so we have refactored tests here